SR2023/2022-10-19-kickstart-and-kit-disclaimer.md

```diff
@@ -0,0 +1,56 @@
+---
+to: Student Robotics 2023
+subject: Getting your kit & Kickstart details
+---
+
+Hello there!
+
+## Kits
+
+We need to organize getting you your kit. Before this can happen, we need you to
+sign a disclaimer. This disclaimer covers the terms and conditions under which
+you may use the kit and outlines the limits of our responsibilities.
+
+As it's a legal document, please print out and sign the disclaimer form
+attached. If you're coming to Kickstart please bring it with you on the day and
+we'll also have spare copies printed out. Otherwise please scan it and email it
+back to us.
+
+For those not coming to Kickstart we will send out your kit to the address
+listed below. Please let us know explicitly if this address is correct, and if
+it's not, let us know an updated one.
+
+Address:
+
+We will need this by the 5th November otherwise your place may be allocated to
+another team. If this is going to be unachievable please let us know and we can
+work something out.
+
+## Kickstart
+
+As you already know, [Kickstart][kickstart] is this Saturday 19th October. For
+those attending in person please arrive at 9:45am at [Building 46][B46] on
+Highfield Campus.
+
+The presentation, which will be livestreamed from there, will start at 10am UK
```

Comment:
Suggestion: We should explicitly say where the livestream link is

Reply:
https://github.com/srobo/team-emails/pull/117/commits/25ea81e5e4ee7770ecd04a1efd82cbbc24240274